src/mm-repeater/mm-repeater.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function(scope) {
+
+	var Validator = StrandLib.Validator,
+		DataUtils = StrandLib.DataUtils;
+
+	scope.Repeater = Polymer({
+		is: 'mm-repeater',
+
+		properties: {
+			config: {
+				type: Object,
+				value: {}
+			},
+			data: {
+				type: Array,
+				notify: true,
+				value: function() { return [{}]; },
+				observer: '_handleDataChanged'
+			},
+
+			template: {
+				type: Object,
+				value: null
+			},
+
+			added: {
+				type: Array,
+				value: []
+			},
+			removed: {
+				type: Array,
+				value: []
+			},
+
+			_origData: {
+				type: Array,
+				value: []
+			},
+
+			addRowLabel: {
+				type: String,
+				value: '+Add Item'
+			}
+		},
+
+		behaviors: [
+			StrandTraits.Refable,
+			StrandTraits.Resolvable
+		],
+
+		get changed() {
+			return this._origData.filter(function(elt) { return elt._changed; });
+		},
+
+		get value() {
+			return this.data;
+		},
+
+		set value(newVal) {
+			if (newVal instanceof Array) {
+				this.set('data', newVal);
+			}
+		},
+
+		observers: [
+			'_handleDataPath(data.*)'
+		],
+
```

Comment:
Add attached/ready with the 'experimental' message `console.warn('This component contains experimental features. The configuration and API are subject to change. Please use at your own risk.');`